sky/provision/__init__.py

```diff
@@ -123,10 +123,17 @@ def delete_volume(provider_name: str,
 
 
 @_route_to_cloud_impl
-def get_volume_usedby(provider_name: str,
-                      config: models.VolumeConfig,
-                      pod_only: bool = False) -> Tuple[List[str], List[str]]:
-    """Get the usedby of a volume."""
+def get_volume_usedby(
+    provider_name: str,
+    config: models.VolumeConfig,
```

Comment:
Instead of `config` should we have a volume handle (`volume_name`, or a `volume_handle` if we have one) instead of a volume config?

Reply:
The `models.VolumeConfig` is the volume handle in code.

Reply:
Updated